src/sdk/src/fragment-components/a-slider.tsx

```diff
@@ -0,0 +1,240 @@
+			})} />
+	</>;
+};
+
+export const ASliderCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+	value={selectedComponent.codeContext?.name}
+	labelText='Input name'
+	onChange={(event: any) => {
+		setComponent({
+			...selectedComponent,
+			codeContext: {
+				...selectedComponent.codeContext,
+				name: event.currentTarget.value
+			}
+		});
+	}}
+/>;
+
+export const ASlider = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		rejectDrop={true}
+		{...rest}>
+			<div className={cx(preventCheckEventStyle, componentObj.cssClasses?.map((cc: any) => cc.id).join(' '))}>
+				<Slider
+				id={componentObj.id}
+				labelText={componentObj.labelText}
+				min={componentObj.min}
+				max={componentObj.max}
+				value={componentObj.value}
+				disabled={componentObj.disabled}
+				step={componentObj.step}
+				hideTextInput={componentObj.textInputIsHidden}
+				stepMultiplier={componentObj.stepMultiplier}
+				minLabel={componentObj.minLabel}
+				maxLabel={componentObj.maxLabel}
+				light={componentObj.light} />
+			</div>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ASlider,
+	settingsUI: ASliderSettingsUI,
+	codeUI: ASliderCodeUI,
+	render: ({ componentObj, select, remove, selected }) => <ASlider
+		componentObj={componentObj}
+		select={select}
+		remove={remove}
+		selected={selected} />,
+	keywords: ['slider'],
+	name: 'Slider',
+	type: 'slider',
+	defaultComponentObj: {
+		type: 'slider',
+		labelText: 'Slider label',
+		min: 0,
+		max: 100
+	},
+	image,
+	codeExport: {
+		angular: {
+			inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Label = "${json.labelText}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Disabled = ${json.disabled ? json.disabled : false};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Value = ${json.value ? json.value : json.min};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Min = ${json.min};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Max = ${json.max};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}MinLabel = "${json.minLabel ? json.minLabel: ''}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}MaxLabel = "${json.maxLabel ? json.maxLabel: ''}";
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Step = ${json.step ? json.step : 1};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}StepMultiplier = ${json.stepMultiplier ? json.stepMultiplier : 1};
+				@Input() ${nameStringToVariableString(json.codeContext?.name)}Theme = "${json.light ? 'light' : 'dark'}" `,
+			outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}ValueChange = new EventEmitter();`,
+			imports: ['SliderModule'],
+			otherImports: () => 'import { CommonModule } from "@angular/common"',
```

Comment:
```suggestion
			otherImports: () => 'import { CommonModule } from "@angular/common";',
```

Reply:
Also, are `otherImports` ever used in Angular?

React export has code that takes it into account. Not sure about Angular export.